DoomEternal.cs

```diff
@@ -21,18 +21,21 @@ public class DoomEternal
 
         public static void EnumerateSaves() {
             Saves = new DoomEternalSaveCollection();
-            foreach(var single in Directory.GetDirectories(BnetSavePath, "*.*", SearchOption.TopDirectoryOnly)) {
-                if(Utilities.CheckUUID(Path.GetFileNameWithoutExtension(single)))
-                    Saves.Add(new DoomEternalSave(Path.GetFileNameWithoutExtension(single), BnetSavePath, DoomEternalSavePlatform.BethesdaNet));
+            if (Directory.Exists(BnetSavePath)) {
+                foreach (var single in Directory.GetDirectories(BnetSavePath, "*.*", SearchOption.TopDirectoryOnly)) {
+                    if (Utilities.CheckUUID(Path.GetFileNameWithoutExtension(single)))
+                        Saves.Add(new DoomEternalSave(Path.GetFileNameWithoutExtension(single), BnetSavePath, DoomEternalSavePlatform.BethesdaNet));
+                }
             }
-            foreach(var steamId3 in Directory.GetDirectories(SteamSavePath, "*.*", SearchOption.TopDirectoryOnly)) {
-                Console.WriteLine(Path.GetFileNameWithoutExtension(steamId3));
-                foreach(var single in Directory.GetDirectories(steamId3, "*.*", SearchOption.TopDirectoryOnly)) {
-                    if (Path.GetFileNameWithoutExtension(single) == SteamGameID.ToString())
-                        Saves.Add(new DoomEternalSave((int.Parse(Path.GetFileNameWithoutExtension(steamId3)) + 76561197960265728).ToString(), SteamSavePath, DoomEternalSavePlatform.Steam));
+            if (Directory.Exists(SteamSavePath)) {
+                foreach (var steamId3 in Directory.GetDirectories(SteamSavePath, "*.*", SearchOption.TopDirectoryOnly)) {
+                    Console.WriteLine(Path.GetFileNameWithoutExtension(steamId3));
+                    foreach (var single in Directory.GetDirectories(steamId3, "*.*", SearchOption.TopDirectoryOnly)) {
+                        if (Path.GetFileNameWithoutExtension(single) == SteamGameID.ToString())
+                            Saves.Add(new DoomEternalSave((int.Parse(Path.GetFileNameWithoutExtension(steamId3)) + 76561197960265728).ToString(), SteamSavePath, DoomEternalSavePlatform.Steam));
+                    }
                 }
             }
-            Saves.Add(new DoomEternalSave("savegame.unencrypted", BnetSavePath, DoomEternalSavePlatform.BethesdaNet));
```

Comment:
If you remove this line savegame.unencrypted will be unsupported.

Reply:
I added that line back in on line 25:
https://github.com/GoobyCorp/DOOMSaveManager/pull/4/files#diff-ae5d80bf40009043e9da3da28dc3b6a6R25